.github/workflows/download_cleanup.yml

```diff
@@ -25,41 +25,31 @@ jobs:
         if: env.SSH_AUTH_SOCK != null
         run: |
           mkdir empty_folder
-          echo 1.12/ > include_file.txt
-          echo 1.12/* >> include_file.txt
-          echo 1.12/*/* >> include_file.txt
-          echo 2.0/ >> include_file.txt
-          echo 2.0/* >> include_file.txt
-          echo 2.0/*/* >> include_file.txt
-          echo 2.1/ >> include_file.txt
-          echo 2.1/* >> include_file.txt
-          echo 2.1/*/* >> include_file.txt
-          echo 2.2/ >> include_file.txt
-          echo 2.2/* >> include_file.txt
-          echo 2.2/*/* >> include_file.txt
+          1.12 >> include_file.txt
+          1.12/release >> include_file.txt
+          1.12/release/* >> include_file.txt
+          1.12/release/*/* >> include_file.txt
+          1.12/manual >> include_file.txt
+          1.12/manual/* >> include_file.txt
+          1.12/manual/*/pdf >> include_file.txt
+          1.12/manual/*/pdf/* >> include_file.txt
+          1.12/manual/*/html >> include_file.txt
+          1.12/manual/*/html/build >> include_file.txt
+          1.12/manual/*/html/build/html >> include_file.txt
+          1.12/manual/*/html/build/html/* >> include_file.txt
+          2.0 >> include_file.txt
+          2.0/release >> include_file.txt
+          2.0/release/* >> include_file.txt
+          2.0/release/*/* >> include_file.txt
+          2.0/manual >> include_file.txt
+          2.0/manual/* >> include_file.txt
+          2.0/manual/*/pdf >> include_file.txt
+          2.0/manual/*/pdf/* >> include_file.txt
+          2.0/manual/*/html >> include_file.txt
+          2.0/manual/*/html/build >> include_file.txt
+          2.0/manual/*/html/build/html >> include_file.txt
+          2.0/manual/*/html/build/html/* >> include_file.txt
```

Comment:
Hmm looks like we need to go down one more level.
https://github.com/mixxxdj/mixxx/actions/runs/8703427569/job/23869574192#step:3:47
Can you check if it works with wildcrads only?
http://downloads.mixxx.org/snapshots/1.12/manual/*/*/*/*